src/templates/dandelion/config/helpers/getBlockTime.js

```diff
@@ -0,0 +1,13 @@
+import { network } from '../../../../environment'
+
+const NETWORK_TIMES = new Map([
+  ['main', 13.5],
+  ['kovan', 4],
+  ['rinkeby', 14.5],
+  ['ropsten', 11.5],
+  ['goerli', 15],
+  ['private', 2],
+])
+export default function getBlockTime() {
+  return network ? NETWORK_TIMES.get(network.type) : null
```

Comment:
The apps work on block numbers instead of timestamps?

Perhaps we should just assume a standard Ethereum target for `main`, `rinkeby`, and `ropsten`, since this is always changing based on the difficulty bomb (I believe it's 12-15s?).